com.unity.render-pipelines.high-definition/Runtime/PostProcessing/Shaders/FinalPass.shader

```diff
@@ -93,7 +93,7 @@ Shader "Hidden/HDRP/FinalPass"
             #if defined(BILINEAR) || defined(CATMULL_ROM_4) || defined(LANCZOS)
             CTYPE outColor = UpscaledResult(positionNDC.xy);
             #elif defined(CONTRASTADAPTIVESHARPEN)
-            CTYPE outColor = LOAD_TEXTURE2D_X(_InputTexture, round(input.texcoord * _ViewPortSize.xy)).CTYPE_SWIZZLE;
+            CTYPE outColor = LOAD_TEXTURE2D_X(_InputTexture, ((input.texcoord.xy * _UVTransform.xy) + _UVTransform.zw) * _ViewPortSize.xy).CTYPE_SWIZZLE;
```

Comment:
Was this introduced with the previous recent PR on CAS or was there before?

Reply:
Was asking for the changelog, but I guess we actually shipped a package since anyway, so worth adding changelog anyway

Reply:
I introduced it with my cas fixes :)

Reply:
Cool, either way given we shipped a package in between we should add a changelog line in fixed

Reply:
Done!